javaparser-core/.gitignore

```diff
@@ -0,0 +1,2 @@
+#The bnd.bnd file is generated by the run_core_generators.sh script (specifically, by the BndGenerator).
+bnd.bnd
```

Comment:
I was a little scared of this because the generators are run by hand. It doesn't happen for every release. So bnd.bnd might be missing for a few releases until I remember to run the generators (but I guess the file will stay around then.)

Reply:
You're right, I've reverted it and added a comment.